Src/DependencyCollector/Net46.Tests/DependencyTrackingTelemetryModuleTestNet46.cs

```diff
@@ -74,6 +86,9 @@ public void TestDependencyCollectionNoParentActivity()
                     -TimeSpan.FromMinutes(1).TotalMilliseconds, 
                     "now - 1 min < timestamp");
                 Assert.AreEqual("200", item.ResultCode);
+                Assert.AreEqual(
+                    SdkVersionHelper.GetExpectedSdkVersion(typeof(DependencyTrackingTelemetryModule), "rdddsd:"),
```

Comment:
This test uses DependencyTrackingTelemetryModule() with the config you created above. The config by default should have EnableDiagnosticSourceInstrumentation as false right? If thats the case, then how did we end up with 'rddsd' as version prefix?